app/src/main/java/com/framgia/englishconversation/screen/videoDetail/VideoDetailViewModel.java

```diff
@@ -59,6 +59,11 @@ public void onItemUserNameClick(TimelineModel item) {
                     mContext.startActivity(
                             ProfileUserActivity.getInstance(mContext, item.getCreatedUser()));
                 }
+
+                @Override
+                public void onItemOptionClick(TimelineModel item) {
+
+                }
```

Comment:
ERROR: 'method def rcurly' have incorrect indentation level 16, expected level should be one of the following: 8, 12.